xinference/core/worker.py

```diff
@@ -793,6 +798,7 @@ async def launch_builtin_model(
             logger.exception(e)
             raise
         try:
+            _ = await self.get_supervisor_ref()
```

Comment:
What is this for?

Reply:
to ensure _event_collector_ref is initialized